coalib/output/printers/LogPrinterMixin.py

```diff
@@ -0,0 +1,45 @@
+from coalib.output.printers.LOG_LEVEL import LOG_LEVEL
```

Comment:
more code, more complexity, IMO remove it if it's not too complicated otherwise schedule for removal later but while we're at this it'd be cool to do a clean job getting rid of all of this.

The only important thing that uses LogPrinterMixin is the Bear because that's API that's used by our users *so* we should properly deprecate that IMO and tell them to use logging instead.